example/ios/Podfile.lock

```diff
@@ -667,6 +661,6 @@ SPEC CHECKSUMS:
   XMTPRust: 2c356f57f034d771717933a6127a7d51667f9843
   Yoga: ba09b6b11e6139e3df8229238aa794205ca6a02a
 
-PODFILE CHECKSUM: c863de1064bef6a085e5c1c74f7ce05cd040466a
+PODFILE CHECKSUM: 522d88edc2d5fac4825e60a121c24abc18983367
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.11.3
```

Comment:
No change needed, but I'm on 1.12.0 and Pat is using 1.12.1 (latest) IIRC. There is a chance that Cocoapod version will matter at some point but shouldn't be an issue now.